presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveStorageFormats.java

```diff
@@ -199,15 +206,75 @@ public void testCreatePartitionedTableAs(StorageFormat storageFormat)
         query(createTableAsSelect);
 
         // SELECT FROM TABLE
-        assertSelect("select sum(tax), sum(discount), sum(length(returnflag)) from %s", tableName);
+        assertSelect("select sum(tax), sum(discount), sum(length(returnflag)) from %s", tableName, "tpch." + TPCH_SCHEMA + ".lineitem");
 
         // DROP TABLE
         query(format("DROP TABLE %s", tableName));
     }
 
-    private static void assertSelect(String query, String tableName)
+    @Requires(ImmutableLineItemTable.class)
+    @Test(groups = {STORAGE_FORMATS, BIG_QUERY})
+    public void testSelectFromPartitionedBzipTable()
     {
-        QueryResult expected = query(format(query, "tpch." + TPCH_SCHEMA + ".lineitem"));
+        String tableName = "storage_formats_test_select_partitioned_bzip";
+
+        // DROP TABLE
+        query(format("DROP TABLE IF EXISTS %s", tableName));
+
+        // CREATE TABLE
+        String createTable = format(
+                "CREATE TABLE %s(" +
+                        "   l_orderkey      BIGINT," +
+                        "   l_partkey       BIGINT," +
+                        "   l_suppkey       BIGINT," +
+                        "   l_linenumber    INT," +
+                        "   l_quantity      DOUBLE," +
+                        "   l_extendedprice DOUBLE," +
+                        "   l_discount      DOUBLE," +
+                        "   l_tax           DOUBLE," +
+                        "   l_linestatus    VARCHAR(1)," +
+                        "   l_shipinstruct  VARCHAR(25)," +
+                        "   l_shipmode      VARCHAR(10)," +
+                        "   l_comment       VARCHAR(44)" +
+                        ") PARTITIONED BY (l_returnflag VARCHAR(1)) ROW FORMAT DELIMITED FIELDS TERMINATED BY '|' STORED AS TEXTFILE",
```

Comment:
it's in setHiveConfigsForBzipInsert -- it isn't a table property, the files stored are just in bz2 format